spek-runtime-test/build.gradle

```diff
@@ -0,0 +1,8 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin.gradle"
+
+dependencies {
+    compile project(':spek-runtime')
+    compile 'org.jetbrains.kotlin:kotlin-stdlib'
+    compile 'org.jetbrains.kotlin:kotlin-reflect'
```

Comment:
that will be covered in `dependencies.gradle`.  Our build is using https://github.com/spring-gradle-plugins/dependency-management-plugin, to manage dependency versions.